_maps/configs/minutemen_corvus

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
It's good practice to add the schema here.

Reply:
where would i find the "schema"?